src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSwitchExpressions.java

```diff
@@ -0,0 +1,114 @@
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.blocks.emptyblock;
+
+/* Config:
+ *
+ * tokens = {LITERAL_WHILE , LITERAL_TRY , LITERAL_CATCH , LITERAL_FINALLY ,
+ *  LITERAL_DO , LITERAL_IF , LITERAL_ELSE , LITERAL_FOR , INSTANCE_INIT ,
+ *  STATIC_INIT , LITERAL_SWITCH , LITERAL_SYNCHRONIZED , LITERAL_CASE ,
+ *  LITERAL_DEFAULT , ARRAY_INIT}
+ * BlockOption = statement
+ */
+public class InputEmptyBlockSwitchExpressions {
+    void howMany1(Nums k) {
+        switch (k) {
+            case ONE:
+                System.out.println("case two");
+            case TWO, THREE: { // violation
+
+            }
+
+            case FOUR: {
+                System.out.println("case three");
+            }
+
+            default:
+                throw new IllegalStateException("Not a nums");
+        }
+    }
+
+    void howMany2(Nums k) {
+        switch (k) {
+            case ONE ->
+                    System.out.println("case one");
+
+            case TWO, THREE ->
```

Comment:
One interesting case is missing here:
```java
            case ONE -> { break; } // OK
            case TWO, THREE -> {} // violation
            case FOUR -> {
               // this block was intentionally left empty
            } // OK when BlockOption == text
```
But I'm not sure about that. The code ` { break; }` looks a bit weird. And we [already have](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockCase.java#L10) an exception for the `switch case`:
```java
        switch (k) {
            case ONE: {} // OK
            case TWO:
              break;
          }
```

This behavior is (poorly) [documented](https://checkstyle.sourceforge.io/config_blocks.html#EmptyBlock), let's leave it as it is. @nmancus1 please add a test case here:
```java
    case TWO, THREE -> {} // OK, empty blocks are allowed for cases
```

Reply:
Done.